### ISSUE TITLE
Stop Viewer

### ISSUE THREAD

**Issue by msrichardson** [conveyal/trimet-mod-otp#118]
**1.** As with the Stop Viewer in the call taker application, please only display departures.

   ![croppercapture 318](https://user-images.githubusercontent.com/4853348/46884313-7f442500-ce09-11e8-82ad-a5b2ca15b83f.png)

   ![croppercapture 319](https://user-images.githubusercontent.com/4853348/46884380-a8fd4c00-ce09-11e8-8ff5-7e254362eb46.png)

**2.** Please always show at least one next trip, even if it is later in the same day, the next day, or even three days later.

   ![croppercapture 316](https://user-images.githubusercontent.com/4853348/46884189-1b216100-ce09-11e8-8ef4-2dafc8cb4040.png)

   ![croppercapture 317](https://user-images.githubusercontent.com/4853348/46884237-43a95b00-ce09-11e8-8899-a0da05089c7a.png)

**Comment by msrichardson** [comment]
@demory I'm finding out.

**Comment by msrichardson** [comment]
@demory Let's do four days (in case Monday is a holiday).

Re: your earlier questions, I think we'll want a countdown, but I'm verifying.  If yes, the cutoff is 59 min in advance.

**Comment by msrichardson** [comment]
(Above has been completed)

Please make the following additional changes to the Stop Viewer:

1. Add ability to refresh TransitTracker
     - At minimum, would be manual, but ideally would allow for both manual and auto-refresh (showing last time updated).

2. Break routes out so that there is one 'tray' per route/destination
     - For example, there are several variations of the eastbound Line 15 - there should be one tray for each (as appropriate and as opposed to current, where customer doesn't know if displayed time is for their destination or not until they open the tray)
      ![CropperCapture 377](https://user-images.githubusercontent.com/4853348/55487052-3b74f000-55e2-11e9-8fe6-826a9af41e0c.png)
--
![CropperCapture 378](https://user-images.githubusercontent.com/4853348/55487280-a9b9b280-55e2-11e9-9f47-ef2901b56292.png)

3. Limit display to next three vehicles per route/destination (via config)

     - Add 'View full schedule' link' to bottom of each tray (example from trimet.org below)
     ![CropperCapture 379](https://user-images.githubusercontent.com/4853348/55487502-12a12a80-55e3-11e9-8e71-8ae4fa9a5fb5.png)

4. Change "0 min" countdown to "Due" (via config file?)

5. If not already an option, allow ability to turn off display of the Status field via config file.

**Comment by msrichardson** [comment]
Moved April 3rd additions to new ticket ( #184).
